javascript/javascript3/week1/Weather/script.js

```diff
@@ -0,0 +1,44 @@
+//Weather API call
+const APIKEY = "9469946e53df5e8174588db17f99cd64";
+const limit = 3;
+
+const country = document.getElementById('search-country');
+const cityName = document.getElementById('search-city');
+const button = document.getElementById('button');
+
+const resultUl = document.getElementById('result-ul');
+
+button.addEventListener('click', (res)=>{
+    res.preventDefault();
+    let weatherCall = new Promise((resolve,reject) =>{
+    fetch(`http://api.openweathermap.org/geo/1.0/direct?q=${cityName.value},${country.value}&limit=${limit}&appid=${APIKEY}`)
```

Comment:
Bug:
What if the user didn't fill anything in before clicking search?
It would also be nice to show some kind of message to indicate that processing is happeing!